ui/src/router.tsx

```diff
@@ -134,19 +152,25 @@ function additionalRoutes() {
           element: <UnderlaySelect />,
         },
         {
-          element: <SourceContextRoot />,
+          element: <StudySourceContextRoot />,
           children: [
             {
               path: "underlays/:underlayName",
-              element: <StudiesList />,
-            },
-            {
-              path: "underlays/:underlayName/studies/:studyId",
-              element: <StudyOverview />,
-            },
-            {
-              path: "underlays/:underlayName/studies/:studyId/*",
-              element: <TanagraContainer />,
+              element: <UnderlaySourceContextRoot />,
```

Comment:
I'm confused why the underlay root is a child of the study root -- isn't the path underlay/study?